### DISCUSSION TITLE
Possible to send one instance of Fooocus to multiple ports/webuis?

### DISCUSSION THREAD

**Discussion by ProgramDeez** [lllyasviel/Fooocus#3022] [Q&A] [answered]
Haven't had any luck with this, but was curious if it is possible? The idea would be to be able to watch the same image generate from multiple ports. If not able, would a viable workaround be to create a 2nd webui instance that is connected to the 1st and uses event listeners to update simultaneously? Appreciate it in advance!

**Comment by mashb1t** [comment, marked as answer]
This is not possible. You would need some kind of websocket proxy like a turn server, but the nature of websockets is that they are end-to-end brokered connections without  the need of a proxy inbetween, only as broker. Let me know if you find anything useful!

**Reply by ProgramDeez** [reply]
Appreciate the insight! Will continue to look into it, but helps knowing that I'm fighting an uphill battle lol
